### PR TITLE
Fix panic in router disconnect when fingerprint is nil. Fixes #1548

### LINKED ISSUE #1548: Panic in edge@v0.24.326/controller/sync_strats/sync_instant.go:194
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x2c6017e]
goroutine 55254237 [running]:
github.com/openziti/edge/controller/sync_strats.(*InstantStrategy).RouterDisconnected(0xc0026a0c00, 0xc002c453b0)
    github.com/openziti/edge@v0.24.326/controller/sync_strats/sync_instant.go:194 +0x31e
github.com/openziti/edge/controller/env.(*Broker).RouterDisconnected.func1()
    github.com/openziti/edge@v0.24.326/controller/env/broker.go:110 +0x36b
created by github.com/openziti/edge/controller/env.(*Broker).RouterDisconnected
    github.com/openziti/edge@v0.24.326/controller/env/broker.go:105 +0x72
```

### PR DESCRIPTION
**Pull request by andrewpmartinez**

### CODE REVIEW

File: controller/sync_strats/sync_instant.go [hunk 1]
```diff
@@ -191,7 +192,7 @@ func (strategy *InstantStrategy) RouterDisconnected(router *network.Router) {
 	log := pfxlog.Logger().WithField("sync_strategy", strategy.Type()).
```
Comment (plorenz) [inline controller/sync_strats/sync_instant.go:192]: This is already fixed. Not sure why it doesn't show up that way.

Comment (andrewpmartinez) [reply controller/sync_strats/sync_instant.go:192]: Weird. I didn't add this.

Comment (andrewpmartinez) [reply controller/sync_strats/sync_instant.go:192]: Also this isn't what I named the PR.

**Comment by andrewpmartinez** [comment]
I'm going to abandon this PR. I'm not sure what is going on here.

**Pull request closed without merging**